File: Question2.py
'''
Write a program to generate the Fibonacci sequence up to 100.
'''

# fn = fn_1 + fn_2
fn_0=0
fn_1=1
fibo_seq=[fn_0,fn_1]

while True:
    next_num = fn_0+fn_1
    if next_num > 100:
        break
    fibo_seq.append(next_num)
    fn_0,fn_1 = fn_1,next_num
print(fibo_seq)

File: Question2.c
//Write a program to generate the Fibonacci sequence up to 100.
# include <stdio.h>

int main(){
    int start,f_0=0;
    int f_1=1;
    int fibo_seq[14];
    fibo_seq[0]=0;
    fibo_seq[1]=1;
    int sum;
    for(start=2;start<14;start++){
        sum =f_0 + f_1;
        if (sum>=100){
            break;
        }
        fibo_seq[start]=sum;
        f_0=f_1;
        f_1=sum;
        printf("%d\n", fibo_seq[start]);
    }
    
}